gpu/surface/src/internal.h

```diff
@@ -37,19 +37,19 @@
 
 #pragma once
 
-#include <pcl/gpu/containers/device_array.h>
+#include <cstdint>
 #include <cuda_runtime.h>
 
+#include <pcl/gpu/containers/device_array.h>
+
 namespace pcl
 {
   namespace device
   {
-	  using std::uint64_type = unsigned long long;
```

Comment:
@PointCloudLibrary/maintainers allow API breakage here since this is an unstable module?

Reply:
Maybe deprecate it, and not use it. Like with `pcl::int_t` types

Reply:
Name of file is `internal.h` - this doesn't sound as it is intended to be used outside.

Reply:
> Maybe deprecate it, and not use it. Like with `pcl::int_t` types

It is relevant to mention, that I have not been able to compile PCLs CUDA code with C++14 flags. So the usual deprecated attribute might be off-limits and we got rid of our custom deprecation pragmas.

Reply:
Putting a line in CHANGELOG (and/or RELEASE_NOTES) should be sufficient since this is an internal API and that is allowed to change without warnings.